exchanges/subscription/list_test.go

```diff
@@ -114,6 +114,17 @@ func TestAssetPairs(t *testing.T) {
 	}
 }
 
+// TestFillAssetPairs exercises AssetPairs handling with no enabled pairs
+// All other code is covered under TestAssetPairs and TestExpandTemplates
+func TestFillAssetPairs(t *testing.T) {
+	e := newMockEx()
+	e.pairs = currency.Pairs{}
+	ap := assetPairs{}
+	err := fillAssetPairs(ap, asset.Spot, e)
```

Comment:
suggestion: any chance we can have fillAssetPairs a method to assetPairs, can resolve if its too much drama.

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/1aba0ee4a